android/res/values-fr/strings.xml

```diff
@@ -530,7 +530,7 @@
 	<!-- Cuisine -->
 	<string name="cuisine_chinese">cuisine chinoise</string>
 	<!-- Cuisine -->
-	<string name="cuisine_coffee_shop">café</string>
+	<string name="cuisine_coffee_shop">Café</string>
```

Comment:
На французском это кофе

> On 18.3.2016, at 15.35, Ilya Zverev <notifications@github.com> wrote:
> 
> In android/res/values-fr/strings.xml:
> 
> > @@ -530,7 +530,7 @@
> >  	<!-- Cuisine -->
> >  	<string name="cuisine_chinese">cuisine chinoise</string>
> >  	<!-- Cuisine -->
> > -	<string name="cuisine_coffee_shop">café</string>
> > +	<string name="cuisine_coffee_shop">Café</string>
> А почему не coffee?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>